src/core/qgsfield.h

```diff
@@ -282,6 +296,18 @@ class CORE_EXPORT QgsField
      */
     void setAlias( const QString &alias );
 
+    /**
+     * Returns the Flags for the field (searchable, …)
```

Comment:
```suggestion
     * Returns the Flags for the field (searchable, ...)
```

I'd skip the unicode characters in dox -- it's likely to open up a world of hurt